mlir/lib/Dialect/Vector/Transforms/VectorEmulateNarrowType.cpp

```diff
@@ -67,6 +75,9 @@ static FailureOr<Operation *> getCompressedMaskOp(OpBuilder &rewriter,
   shape.back() = numElements;
   auto newMaskType = VectorType::get(shape, rewriter.getI1Type());
   if (createMaskOp) {
+    if (intraVectorOffset != 0) {
+      return failure();
+    }
```

Comment:
few nits:

1. Remove the braces for simple single statement.
2. Add some comments (perhaps TODO) to explain why it is not supported.

My guess is that they could be dynamic values, so it is tricky to generate the mask? Given that we're going to support dynamic offsets in a follow-up, it looks okay to me. Do you have any ideas about how to support it in mind yet? My initial idea is that we could create `rank+1` masks using `CreateMaskOp` and a sequence of bitwise operations to construct the mask. I'm happy to chat when we're adding the support.